ci/teams/rhtap-build/release.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleasePlan
+metadata:
+  name: build-release-to-quay
+  namespace: rhtap-build-tenant
+  labels:
+    release.appstudio.openshift.io/auto-release: 'true'
```

Comment:
we also need this:

`release.appstudio.openshift.io/standing-attribution: 'true'`

Reply:
still missing

Reply:
added 👍 Another round of review, please?